src/main/kotlin/uk/gov/justice/digital/hmpps/hmppschallengesupportinterventionplanapi/model/request/UpdateContributoryFactorRequest.kt

```diff
@@ -1,14 +1,9 @@
 package uk.gov.justice.digital.hmpps.hmppschallengesupportinterventionplanapi.model.request
 
 import io.swagger.v3.oas.annotations.media.Schema
-import jakarta.validation.constraints.Size
 
 @Schema(description = "The request body to update a contributory factor to the incident that motivated the CSIP referral")
 data class UpdateContributoryFactorRequest(
-  @Schema(description = "The type of contributory factor to the incident or motivation for CSIP referral.")
-  @field:Size(min = 1, max = 12, message = "Contributory factor type code must be <= 12 characters")
-  override val factorTypeCode: String,
-
   @Schema(description = "Additional information about the contributory factor to the incident or motivation for CSIP referral.")
   override val comment: String?,
```

Comment:
This wasn't in the ticket so not a missed AC. There's a 4k char limit on the CF comments

Reply:
Ah it's cause it's 4k chars in NOMIS so we're limiting it (for now) for sync. I didn't realise it wasn't on the create. I think that's just been missed as the 4k limit is on other text fields.